src/pages/Tenants.tsx

```diff
@@ -51,9 +66,16 @@ const Tenants: React.FC<TenantsProps> = ({ selectedView }) => {
             headers={headers}
             hasIcon={true}
             hasButtons={true}
-            tenants={tenants}
+            tenants={currentTenants}
             setTenants={setTenants}
           />
+          <Pagination
+            data={tenants}
```

Comment:
I think you are passing the tenants only for getting count. So, instead of passing tenants you can pass the count of tenants